vtexdocs/devportal
# Feature/On this page

#### What is the purpose of this pull request?

To add the mobile versions of the contributors component and the table of contents.

Documented [here](https://www.notion.so/vtexhandbook/Ajustar-componente-de-contribuidores-da-p-gina-de-API-Guides-no-breakpoint-de-1024px-ff85bf6244744a2fa0dddf37f3142476)

#### What problem is this solving?

On the first four breakpoints, the contributors component and the table of contents weren't visible. This PR fixes that by positioning the contributors component at the bottom of the page and adding a floating component with the table of contents inside, in these resolutions, as per [Figma](https://www.figma.com/file/Lx6sXdrw9KEvtSEKWttmv8/Developer-Portal?node-id=6005%3A350485).

#### How should this be manually tested?

Go to [this page](https://deploy-preview-63--elated-hoover-5c29bf.netlify.app/docs/api-guides/developing-an-app) and try different breakpoints.

- Make sure the contributors component appears at the bottom of the page on breakpoints 640px through 1279px
- Make sure the new OnThisPage component appears as a floating button on the bottom right corner of the page on breakpoints 360px through 1279px
- Check the layout and behavior of the new OnThisPage component against [Figma](https://www.figma.com/file/Lx6sXdrw9KEvtSEKWttmv8/Developer-Portal?node-id=2701%3A187371)
    - The layout and behavior of the component on the first breakpoint still needs improvement, a new task will be created to fix it
- Check the layout and behavior of the old contributors and table of contents components on higher resolutions (1280px onwards) against [Figma](https://www.figma.com/file/Lx6sXdrw9KEvtSEKWttmv8/Developer-Portal?node-id=6005%3A356875)

#### Screenshots or example usage

https://user-images.githubusercontent.com/37647055/182477911-d5cc34e5-ce94-48fb-8296-8990ca91b3f6.mp4

#### Types of changes
